audits/manifest/exists.js

```diff
@@ -47,7 +47,9 @@ class ManifestExists extends Audit {
    */
   static audit(artifacts) {
     return ManifestExists.generateAuditResult(
-      typeof artifacts.manifest !== 'undefined'
+      typeof artifacts.manifest.value !== 'undefined',
+      undefined,
+      artifacts.manifest.warning
```

Comment:
i got confused and thought `warning` was diff than `errorString`. If they're the same, let's keep it on the same name throughout.